kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/system/SystemServiceImpl.java

```diff
@@ -179,7 +179,7 @@ public Object run() {
             // if not found, look for such file in the kura.home directory
             Properties kuraCustomProps = new Properties();
             String kuraCustomConfig = System.getProperty(KURA_CUSTOM_CONFIG);
-            String kuraCustomProperties = IOUtil.readResource(KURA_CUSTOM_PROPS_FILE);
+            String kuraCustomProperties = readResource(KURA_CUSTOM_PROPS_FILE);
```

Comment:
The point here is testability. If static import is used, the call to static method still cannot be avoided. If it is wrapped in a protected method, the test can bypass it and maybe test a few conditions that are hard to reach with the live method.